src/components/ExerciseHeader.tsx

```diff
@@ -65,13 +65,20 @@ const ExerciseHeader = ({
   const progressText = showProgress ? `${currentWord + 1} / ${numberOfWords}` : ''
 
   useEffect(() => {
-    const renderHeaderLeft = () => (
-      <NavigationHeaderLeft
-        title={labelOverride ?? getLabels().general.header.cancelExercise}
-        onPress={confirmClose ? () => setIsModalVisible(true) : () => navigation.dispatch(closeExerciseAction)}
-        isCloseButton={isCloseButton}
-      />
-    )
+    const renderHeaderLeft = () =>
```

Comment:
This will also change the behavior for other exercises e.g. the choice exercise. But this is not wanted, because, if you go to the last word in the choice exercise, this exercises is not yet finished, you need to enter a solution for the last word to finish this exercise. Therefore the changes you made here should only be displayed for the word list.

Next steps:
Pass a Prop named exerciseKey to the ExerciseHeader component and only apply you changes, if the exerciseKey is  vocabularyList. See data.ts file, where we have defined the ExerciseKeys.